### PR TITLE
[plugin.video.gamekings] 1.2.16

### PR DESCRIPTION
**Pull request by skipmodea1**
v1.2.16 (2019-09-15)
- added premium video category
- fixes due to website change (september 2019). Let's hope they keep working
- added maximum video quality setting for the new m3u8 videos
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.gamekings/resources/lib/gamekings_main.py [hunk 1]
```diff
@@ -27,6 +27,18 @@ def __init__(self):
         # Get the plugin handle as an integer number
         self.plugin_handle = int(sys.argv[1])
 
+        #
+        # Premium Videos
+        #
+        parameters = {"action": "list", "plugin_category": LANGUAGE(30005),
+                      "url": BASE_URL_GAMEKINGS_TV + "category/premium/page/001/", "next_page_possible": "True"}
+        url = self.plugin_url + '?' + urllib.parse.urlencode(parameters)
+        list_item = xbmcgui.ListItem(LANGUAGE(30005), iconImage="DefaultFolder.png")
```
Comment (enen92) [inline plugin.video.gamekings/resources/lib/gamekings_main.py]: Remove iconimage kwarg

Comment (enen92) [reply plugin.video.gamekings/resources/lib/gamekings_main.py]: Add it to setArt as "thumb" or "icon"

**Comment by enen92** [comment]
I assume you tested this on the python3 branch?

File: plugin.video.gamekings/resources/lib/gamekings_main.py (hunk 1 above)
Comment (skipmodea1) [reply plugin.video.gamekings/resources/lib/gamekings_main.py]: Fixed.
Also fixed some other stuff:
- fix for trailers
- removed http:// urls

**Comment by skipmodea1** [comment]
Yeah, i tested it on python 2 and 3

**Pull request merged**